code/handlers/trackservers.q

```diff
@@ -177,11 +174,16 @@ autodiscovery:{if[DISCOVERYRETRY>0; .servers.retrydiscovery[]]}
 
 // Attempt to make a connection for specified row ids
 retryrows:{[rows]
+    //function a checks if the handle passed is empty and also invokes checknontorqattr function 
+    //which checks if .proc.getattributes is defined on the nontorqprocess and executes it 
+    //only if it is defined
+    a:{$[(not null x);@[x;"$[`getattributes in key`.proc;.proc.getattributes[];()!()]";()!()];()!()]};
```

Comment:
Same comment as line 143 may apply here + don't need the brackets round `not null x`.

Reply:
I am trying to avoid errors being raised in the nontorq process. from what I know the apply function will still try to execute the function.

Reply:
Okay, so what I said earlier wouldn't work at all, but if you wanted a way to change that then this might work:
```
@[value;`.proc.getattributes;{::}]()!()
```